GitHubRepositories.ps1

```diff
@@ -1031,7 +1039,12 @@ filter Set-GitHubRepository
 
     .PARAMETER Private
         Specify this to make the repository private.
-        To change a repository to be public, specify -Private:$false
+        To change a repository to be public, specify -Private:$false or use the Visibility
+        parameter.
+
+    .PARAMETER Visibility
+        Specify this to make the repository public, private or internal.  This overrides the
+        private parameter when you use both parameters.
```

Comment:
```suggestion
        Can be public or private. If your organization is associated with an enterprise account
        using GitHub Enterprise Cloud or GitHub Enterprise Server 2.20+, visibility can also be
        internal.

        This will override the Private parameter if also specified in the same command.
```